stl/inc/format

```diff
@@ -1415,21 +1424,23 @@ struct _Format_arg_traits {
     // set of basic_format_arg (N4885 [format.arg]). They determine the mapping
     // from "raw" to "erased" argument type for _Format_arg_store.
     template <_Has_formatter<_Context> _Ty>
-    static auto _Phony_basic_format_arg_constructor(const _Ty&) {
+    static auto _Phony_basic_format_arg_constructor(_Ty&&) {
+        // LWG-3631

```

Comment:
Comment nitpick: We conventionally say something like "per the proposed resolution of LWG-nnnn" to provide a bit more context.